proto/p4/v1/p4runtime.proto

```diff
@@ -284,6 +284,19 @@ message Action {
 
 message ActionProfileActionSet {
   repeated ActionProfileAction action_profile_actions = 1;
+  // Support per-group hash modes and resouce usage modes. Added in v1.5.0.
```

Comment:
Do you have target devices in mind that support independent values of bucket_selection_mode and size_semantics for each ActionProfileActionSet?

That is, suppose table t1 has an action selector implementation, and for its corresponding action selector implementation (call it as1), you create 5 groups.  Do you expect to be able the values of bucket_selection_mode and size_semantics independently for each of those 5 groups?